### PR TITLE
Functions for python3 virtualenv and using nfs mounts

### PR DESCRIPTION
**Pull request by hiyer**

### CODE REVIEW

File: hadoop/util.sh [hunk 1]
```diff
@@ -2,26 +2,49 @@
 
 source /usr/lib/hustler/bin/qubole-bash-lib.sh
 export PROFILE_FILE=${PROFILE_FILE:-/etc/profile}
-export HADOOP_ETC_DIR=${HADOOP_ETC_DIR:-/etc/hadoop}
+export HADOOP_ETC_DIR=${HADOOP_ETC_DIR:-/usr/lib/hadoop2/etc/hadoop}
```
Comment (hiyer) [inline hadoop/util.sh]: Not sure why this file is showing up - there is actually no difference in content.

File: misc/mount_nfs.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# TODO: add instructions for Azure file share
+# 
+
+function mount_nfs_volume() {
+  nfs_export=$1
+  mountpoint=$2
+
+  is_master=$(nodeinfo is_master)
+  if [[ $is_master == "1" ]]; then
+    mount -v -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 "$nfs_export" "$mountpoint"
```
Comment (jsensarma) [inline misc/mount_nfs.sh:24]: isn't the read-writable on master and readable on slave a specific pattern that is not going to be common to all applications? (for example - just mounting a NFS share with some reference data that's used by all nodes?)

File: misc/python_venv.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# Alternatively you can also use a requirements file. For example
+# to use a requirements file stored in S3 or Azure Blob Store, run
+#
+#    /usr/lib/hadoop2/bin/hadoop dfs -get {s3|wasb}://path/to/requirements/file /tmp/requirements.txt
+#    pip install -r /tmp/requirements.txt
+#
+
+function install_python_venv() {
+  version=${$1:-36}
+  location=${$2:-/usr/lib/virtualenv/py36}
```
Comment (jsensarma) [inline misc/python_venv.sh:16]: I am told python 3.5 is already pre-installed on QDS AMIs at this point (at /usr/lib/a-4.2.0-py-3.5.3/). Can we use that instead as the default? (and only yum install versions other than that)

**Review by jsensarma — COMMENTED** [review]
still reviewing - submitting some comments.

File: misc/mount_nfs.sh (hunk 2 above)
Comment (hiyer) [reply misc/mount_nfs.sh:24]: Yes, but users can use this as a template to write a different script where it's writable everywhere.

File: misc/python_venv.sh (hunk 3 above)
Comment (hiyer) [reply misc/python_venv.sh:16]: I believe that's the anaconda version. Should we use that or the system one? The anaconda location is liable to change if we upgrade anaconda, whereas /usr/lib/python3[5|6] will be a constant location.

**Pull request merged**